backend/api_handler.py

```diff
@@ -23,6 +25,7 @@
 for name in ['boto3', 's3transfer', 'botocore', 'boto']:
     logging.getLogger(name).setLevel(logging.ERROR)
 
+load_modules(modes=[ImportMode.API])
```

Comment:
[Clarification] Here is where the magic occurs, we import the modules `api` mode submodules